src/ess/powder/conversion.py

```diff
@@ -229,8 +269,10 @@ def convert_monitor_do_wavelength(
 
 
 providers = (
+    assemble_output_calibration,
     powder_coordinate_transformation_graph,
     add_scattering_coordinates_from_positions,
     convert_to_dspacing,
+    convert_to_calibrated_tof,
```

Comment:
'calibrated' might be misleading. It could be replaced by 'reduced', 'derived', 'final'

Reply:
Better now?